controllers/actions.github.com/autoscalinglistener_controller.go

```diff
@@ -306,6 +306,38 @@ func (r *AutoscalingListenerReconciler) cleanupResources(ctx context.Context, au
 		logger.Info("Listener proxy secret is deleted")
 	}
 
+	listenerRoleBinding := new(rbacv1.RoleBinding)
+	err = r.Get(ctx, types.NamespacedName{Namespace: autoscalingListener.Spec.AutoscalingRunnerSetNamespace, Name: scaleSetListenerRoleName(autoscalingListener)}, listenerRoleBinding)
```

Comment:
How do we guarantee that we're only deleting the roles we created? What if the end user installed the resources through some other mechanism?

Reply:
These roles are created by the controller here: https://github.com/actions/actions-runner-controller/blob/ba91c183b505dfc7a619e0e56db2bf268ff3d569/controllers/actions.github.com/autoscalinglistener_controller.go#L179

We just never did a cleanup on them